components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -1124,6 +1132,7 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf2d, cam_in, cam_out,
                crm_output%clhgh, crm_output%clmed, crm_output%cllow, &
                crm_output%sltend, crm_output%qltend, crm_output%qcltend, crm_output%qiltend, &
                crm_output%t_vt_tend, crm_output%q_vt_tend, crm_output%t_vt_ls, crm_output%q_vt_ls, &
+               crm_input%u_vt, crm_output%u_vt_tend, crm_output%u_vt_ls, &
```

Comment:
you can move 'crm_input%u_vt' next to 'crm_input%t_vt, crm_input%q_vt'

Reply:
done.